oak-store-spi/src/main/java/org/apache/jackrabbit/oak/plugins/memory/AbstractBlob.java

```diff
@@ -189,4 +182,39 @@ public String toString() {
         return getSha256().toString();
     }
 
+    public static class HashCode {
```

Comment:
Could you please revert this change. I don't think this is part of this PR.

Reply:
Indeed; thanks.